cider/tests/utils/DuckDbQueryRunner.cpp

```diff
@@ -532,7 +532,25 @@ DuckDbResultConvertor::fetchDataToArrowFormattedCiderBatch(
     auto chunk = result->Fetch();
     if (!chunk || (chunk->size() == 0)) {
       if (batch_res.empty()) {
-        batch_res.push_back(std::make_shared<CiderBatch>());
+        auto arrow_array = CiderBatchUtils::allocateArrowArray();
+        auto arrow_schema = CiderBatchUtils::allocateArrowSchema();
+
+        arrow_array->length = 0;
```

Comment:
these are same to `CiderBatchUtils::allocateArrowArray`?

Reply:
redundant code removed.